tests/em_test_helper.rb

```diff
@@ -1,8 +1,11 @@
+require 'em/pure_ruby' if ENV['EM_PURE_RUBY']
```

Comment:
Yep :+1: Just wanted an easy way to spot test, as I am sure the current pure ruby reactor will fail many of the tests in the suite.